client/src/controllers/CountController.ts

```diff
@@ -0,0 +1,55 @@
+import { Controller } from '@hotwired/stimulus';
+import { ngettext } from '../utils/gettext';
+
+export class CountController extends Controller<HTMLFormElement> {
```

Comment:
```suggestion
/**
 * Adds the ability for a controlled element to update the total count
 * of selected elements within the provided container selector, defaults
 * do `body.`
 *
 * @example
 * <div data-controller="w-count">
 *  <span data-w-count-target="label"></span>
 *  <span class="error-message">An error</span>
 * </div>
 */
export class CountController extends Controller<HTMLFormElement> {
```

Suggest adding a JSDoc at the start with a basic example.